stories/Input/SelectStory.tsx

```diff
@@ -138,8 +136,14 @@ const props = {
 
 const Story = (
   <div>
+    <div style={{ marginBottom: '1rem' }}>2 sizes only</div>
```

Comment:
I think `2 types only` will be more suitable for this section 🤔

Reply:
but it's sizes

Reply:
Can have another section called `sizes`?

Reply:
Would you mind to check the button story on the branch `feature/GDS-button`?

I'm trying to introduce Button props.

Reply:
prefer do these things separately.